PerfTools/AllocMonitor/plugins/moduleAlloc_setupFile.cc

```diff
@@ -495,6 +498,7 @@ namespace edm::service::moduleAlloc {
                                 esModuleTypesPtr,
                                 moduleCtrDtrPtr,
                                 sourceCtrPtr,
+                                sourceTypesPtr,
```

Comment:
```suggestion
                                sourceTypesPtr=std::move(sourceTypesPtr),
```
would make the intended lifetime of `sourceTypesPtr` more clear.